1_fetch/out/.gitignore

```diff
@@ -0,0 +1 @@
+model_RMSes.csv
```

Comment:
This wouldn't affect how your code runs at all, but as a convention, we typically populate a single `.gitignore` file in the root directory. In this case, we'd just add a line `*/out/*` to ignore everything within any `out/` subdirectory of each phase. It's nice to be able to see in a single place exactly what files across the whole pipeline we're telling git to ignore.

Reply:
Oh, that makes sense! Thanks. I've made the necessary changes.